init.c

```diff
@@ -376,6 +383,8 @@ static int setup_thread_area(void)
 #endif
 
 	memset(threads, 0, max_jobs * sizeof(struct thread_data));
+	for (i = 0; i < max_jobs; i++)
+		DRD_IGNORE_VAR(threads[i]);
```

Comment:
I think if we teach fio to stop detaching threads and join on them when they're complete a lot of the issues related to this one will go away...

Reply:
...but there's an argument that perhaps we should keep this ignore until that happens.